admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -2,117 +2,98 @@
-
-ownCloud's server-side encryption generates a strong encryption key, which is 
-unlocked by user's passwords. Your users don't need to track an extra 
-password, but simply log in as they normally do. It encrypts only the contents 
-of files, and not filenames and directory structures.
-
-You should regularly backup all encryption keys to prevent permanent data loss. 
+For maximum security make sure to enable "Check for changes: Never." when configuring external storage. 
+With it enabled, ownCloud ignores new files not added via ownCloud, and a malicious external storage administrator could not add new files to the storage without your knowledge. 
+Of course, this is not wise if your external storage is subject to legitimate external changes.
+
+Encryption and decryption are performed on the ownCloud server. 
+ownCloud server-side encryption encrypts files stored on the ownCloud server and files on remote storage that are connected to your ownCloud server. 
+All files stored on remote storages will be decrypted upon retrieval, before serving them to you and anyone you have shared them with.
+
+.. note:: Encrypting files increases their size by roughly 35%, so you must take this into account when you are provisioning storage and setting storage quotas. User's quotas are based on the unencrypted file size, and not the encrypted file size.
+
+When files on external storage are encrypted in ownCloud, you cannot share them directly from the external storage services, only through ownCloud sharing, because the key to decrypt the data never leaves the ownCloud server.
+
+ownCloud's server-side encryption generates a strong encryption key, which is unlocked by the user's passwords. 
+Your users don't need to track an extra password, just log in as they normally do. 
+It encrypts only the contents of files, and not filenames and directory structures.
+
+.. important:: 
+   You should regularly backup all encryption keys to prevent permanent data loss. 
+
 The encryption keys are stored in the following directories:
 
-``data/<user>/files_encryption`` 
-  Users' private keys and all other keys necessary to decrypt the users' files
-``data/files_encryption``
-  private keys and all other keys necessary to decrypt the files stored on a
-  system wide external storage
+================================ ==============================================
+Directory                        Description
+================================ ==============================================
+``data/<user>/files_encryption`` Users' private keys and all other keys 
+                                 necessary to decrypt the users' files.
+``data/files_encryption``        Private keys and all other keys necessary to 
+                                 decrypt the files stored on a system wide 
+                                 external storage.
+================================ ==============================================
```

Comment:
After having a look at the file again, it would seem to be a better choice to add sub-headers for the `occ` commands and then link to the relevant command.